notebooks/tr/agents.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<ul><li>Araca bir öznitelik olarak bir vectordb eklememiz gerektiğinden,&nbsp;<code>@tool</code>&nbsp;dekoratörüyle&nbsp;<a href="https://huggingface.co/docs/transformers/main/en/agents#create-a-new-tool" target="_blank">basit araç oluşturucuyu</a>&nbsp;kullanamayız, bu yüzden&nbsp;<a href="https://huggingface.co/docs/transformers/main/en/agents_advanced#directly-define-a-tool-by-subclassing-tool-and-share-it-to-the-hub" target="_blank">gelişmiş ajanlar dökümantasyonunda</a>&nbsp;vurgulanan gelişmiş kurulumu takip edeceğiz.</li></ul><p>burayı daha akıcı/okunabilir şekilde yeniden yazabilir miyiz?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>